app/Models/Foundation/Summit/Registration/PromoCodes/Traits/SpeakersPromoCodeTrait.php

```diff
@@ -240,6 +240,14 @@ public function getQuantityAvailable(): int
         return $this->owners->count();
     }
 
+    /**
+     * @return int
+     */
+    public function getQuantityRemaining(): int
```

Comment:
this is not necessary see https://github.com/OpenStackweb/summit-api/pull/211/files#r1388232821